Sobel/makefile

```
@@ -0,0 +1,11 @@
+CC = gcc
+CFLAGS = -fsanitize=address
```

I recommend adding couple more flags, FYI: https://developers.redhat.com/blog/2018/03/21/compiler-and-linker-flags-gcc
```suggestion
CFLAGS = -Wall -Wextra -Wcast-allign -fstack-protector -fsanitize=address
```